VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json

```diff
@@ -673,7 +673,8 @@
     "leave-hearing": "Leave Hearing",
     "share-video-evidence": "Share video or audio",
     "share-document-evidence": "Share documents",
-    "unsupported-dynamic-evidence": "Video sharing is not supported on this browser"
+    "unsupported-dynamic-evidence": "Video sharing is not supported on this browser",
+    "change-hearing-layout": "Change Layout"
```

Comment:
can we add a welsh translation for this? Or a placeholder one in cy.json (written in English), if we don't have one yet